install.sh

```diff
@@ -255,5 +255,11 @@ run $BUILD/CodeMirror git checkout tags/5.43.0
 run $BUILD/CodeMirror npm install
 run $BUILD/CodeMirror npm install -s uglify-js git+ssh://git@github.com:angelozerr/CodeMirror-Extension.git
 
+# Install pegjs
+
+run $BUILD       git clone https://github.com/pegjs/pegjs.git
```

Comment:
You should be able to install directly using npm instead of git.

Reply:
Yes, I changed this.